extract-coderabbit-feedback.py

```
@@ -372,334 +498,447 @@ def extract_prompt_for_ai_agents(body: str) -> List[str]:
+
     if debug:
-        print(f"DEBUG: parse_file_level_comments found {len(file_matches)} file sections", file=sys.stderr)
-    
+        print(
+            f"DEBUG: parse_file_level_comments found {len(file_matches)} file sections",
+            file=sys.stderr,
+        )
+
     for file_match in file_matches:
         file_path = file_match.group(1).strip()
         comment_count = int(file_match.group(2))
```

Ruff (0.14.4)

Local variable `comment_count` is assigned to but never used

Remove assignment to unused variable `comment_count`

(F841)